### DISCUSSION TITLE
Is there any way to generate longer pieces of audio?

### DISCUSSION THREAD

**Discussion by BlueNebulaDev** [suno-ai/bark#451] [Q&A]
I would like to generate speech from text, but the max length of ~13-14 seconds makes Bark unsuitable for my needs.

Is there any way, or any plans for the future, to generate longer pieces of audio?

**Comment by BlueNebulaDev** [comment]
I look again at the doc and found this, which answers my question: https://github.com/suno-ai/bark#-generating-longer-audio

Maybe you may want to put a link to it in the FAQs: https://github.com/suno-ai/bark#why-is-the-output-limited-to-13-14-seconds

**Comment by lahiru1k** [comment]
Try this fork by @JonathanFly : https://github.com/JonathanFly/bark#generating-longer-audio.

**Comment by donaldafeith** [comment]
I can generate up to a minute or more by altering python files. Although, the longer the audio... It sometimes pauses more and puts in more ums and noises.
